Dockerfile.envoy

```diff
@@ -1,6 +1,6 @@
 FROM debian:sid as builder
 
-ADD qat1.7.l.4.3.0-00033.tar.gz .
```

Comment:
This version seems to be availabe:

https://01.org/sites/default/files/downloads/intelr-quickassist-technology/qat1.7.l.4.3.0-00033.tar.gz